src/OrchardCore.Modules/OrchardCore.Users/Controllers/SmsAuthenticatorController.cs

```diff
@@ -207,7 +206,10 @@ public async Task<IActionResult> ValidateCode(EnableSmsAuthenticatorViewModel mo
         return View(model);
     }
 
-    [HttpPost, Produces("application/json"), AllowAnonymous]
+    // TODO: move this action into minimal API.
```

Comment:
Rather open an issue about this, please.

Reply:
same as above.

Reply:
the comment and the action was removed from the controller :)